lightning/backend/local.go

```diff
@@ -1613,13 +1680,20 @@ func (w *LocalWriter) writeRowsLoop() {
 			w.writeErr.Set(err)
 			return
 		}
-		if err := w.local.db.Ingest([]string{filePath}); err != nil {
-			w.writeErr.Set(err)
-			return
-		}
+		w.local.lockForImport(importMutexStateFlush)
```

Comment:
`lockForImport` will set isImporting to true, thus this will impact the force importing logic since force importing will skip engine in importing state. I think it's better to just lock but not set the atomic var

Reply:
Why? I do not think `force importing` shall choose one db which is ingesting a file. This duration is short and if `force importing` want to choose this db, it shall wait it to finish ingesting.

Reply:
IMO importing atomic var is used to "take a look at it" to decide if it could be further force importing, when further force importing, lightning will do aquire the lock.

so setting importing atomic var in this write phase will cause a "false positive" (I'll check my logic later 😂 )

Reply:
If it meet a `importMutexStateFlush`, it will skip forcing this engineFile importing. This is a very unlikely event. Even if it happens, we just left it force-import in next time.

Reply:
I have change it into another state `NoLock`.